includes/class-api.php

```diff
@@ -153,6 +153,20 @@ public function get_addons() {
 			];
 		}
 
+		// Catch responses that don't contain an array of add-ons.
+		if ( ! isset( $response['addons'] ) ) {
+			return [
+				'addons' => [],
```

Comment:
Travis currently only tests 7.0+, so it's possible we'll miss issues from 5.4, 5.5, and 5.6.

Reply:
Added #22 to address.